public/video-ui/src/components/VideoYoutube/VideoYoutube.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { ManagedForm, ManagedField } from '../ManagedForm';
+import SelectBox from '../FormFields/SelectBox';
+
+export default class YoutubeMetaData extends React.Component {
```

Comment:
I think `YouTubeMetaData` is a clearer name than `VideoYoutube`, could we rename the file?